Cambios en docker practica 3

plss12 opened the pull request:
Realizados los cambios necesarios para el funcionamiento de docker-compose

plss12 committed:
Cambios en docker practica 3

Pabalvcar reviewed (changes requested):
Hay un error en el Dockerfile, en la linea 13 deben cambiar el repositorio de su grupo por el general

TheNeoStormZ commented on docker/Dockerfile:
```diff
@@ -10,7 +10,7 @@ RUN pip install ipython
 WORKDIR /app
 
-RUN git clone https://github.com/wadobo/decide.git .
+RUN git clone https://github.com/Villanueva-del-Trabuco-EGC/decide-part-2.git .
```
Cambiar este enlace al repo comun

plss12 replied:
Lo arreglo

plss12 committed:
Cambio error dockerfile
Cambiado la url del repositorio en el dockerfile para usar el repositorio principal
